website/docs/plugins/ray_launcher.md

```diff
@@ -48,64 +48,61 @@ $ python my_app.py hydra/launcher=ray_aws --cfg hydra -p hydra.launcher
 # @package hydra.launcher
 _target_: hydra_plugins.hydra_ray_launcher.ray_aws_launcher.RayAWSLauncher
 mandatory_install:
```

Comment:
mandatory_install is not a great name.
maybe `node_setup` or even just `setup`?